client/state/followers/test/selectors.js

```diff
@@ -57,7 +49,61 @@ describe( 'selctors', () => {
 				},
 			};
 			const query = { siteId: 555 };
-			expect( getFollowersForQuery( state, query ) ).to.be.null;
+			expect( getFollowersForQuery( state, query ) ).toBeNull;
```

Comment:
Did you mean to invoke it?
```suggestion
			expect( getFollowersForQuery( state, query ) ).toBeNull();
```